languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -63936,6 +63936,63 @@ USA
                 <example>I <marker>haven't said no</marker> yet.</example>
             </rule>
         </rulegroup>
+        <rulegroup id="DOUBLE_NEG_RP" name="I didn't mention it to no one" default="temp_off">
+            <antipattern>
+                <token regexp="yes">could|would|should</token>
+                <token regexp="yes">not|n't</token>
```

Comment:
Did you know? This is totally correct, but you could have written: `<token inflected="yes">not</token>`